src/annotator/config/index.ts

```diff
@@ -207,6 +207,10 @@ const configDefinitions: ConfigDefinitionMap = {
     defaultValue: null,
     getValue: getHostPageSetting,
   },
+  sideBySide: {
+    allowInBrowserExt: true,
```

Comment:
I'm not sure if this option should be allowed in the browser extension or not. What's usually the logic to decide that?

Reply:
We try to make a judgement call about which approach (ignoring or respecting the configuration) is likely to be most useful to a user of the browser extension. The intent of this configuration is about allowing websites to provide a better implementation of side-by-side than the default, so it probably does make sense to allow in the extension.